tests/unit/models/cart-model-test.js

```diff
@@ -96,6 +96,29 @@ test('it creates a line item when you add a variant', function (assert) {
   });
 });
 
+test('it creates a line item when you add a variant', function (assert) {
+  assert.expect(3);
+
+  const done = assert.async();
+
+  const quantity = 2;
+
+  const variant = singleProductFixture.product_listing.variants[1];
+
```

Comment:
Hmmm.. Ok. I'll take a look at the code for this test. This test was actually from `master`. I just literally just copied and pasted it to ensure the old naming of the function still worked and so did the new. (running the same tests with both functions)